community/server/src/main/java/org/neo4j/server/configuration/ConfigurationBuilder.java

```diff
@@ -137,4 +141,13 @@ public Configuration configuration()
         }
 
     }
+
+    class DefaultSettingsClassesHolder
```

Comment:
I also feels a bit strange to keep the static method in a class here. Though I have no better idea where it should go, maybe Jake has some good idea? @jakewins